lib/itamae/runner.rb

```diff
@@ -30,12 +30,17 @@ def initialize(backend, options)
       prepare_handler
 
       @node = create_node
-      @tmpdir = options[:tmp_dir]
+      @tmpdir = options[:tmp_dir] unless options[:tmp_dir].nil?
```

Comment:
I thought that. But, yes that’s right. Then we can remove the below `if` guard, right? 😉 

```suggestion
      @tmpdir = options[:tmp_dir] || "/tmp/itame_tmp"
```

Reply:
@unasuke like this?

https://github.com/itamae-kitchen/itamae/pull/341/commits/c48a649e4bd8ab80e7ef1426d9970055f5e8a9df